common/kvstore/KeyValueStore.cc

```diff
@@ -35,6 +35,12 @@ static void throw_mdb_error(string_view what, int err) {
 // http://www.lmdb.tech/doc/group__mdb.html#gac08cad5b096925642ca359a6d6f0562a
 // That function is called in OwnedKeyValueStore.
 static atomic<bool> kvstoreInUse = false;
+
+int mdbReaderListCallback(const char *msg, void *ctx) {
```

Comment:
What does this do? can we document it?

Reply:
Sure!

LMDB only has two APIs to grok the reader list (http://www.lmdb.tech/doc/group__mdb.html#ga8550000cd0501a44f57ee6dff0188744)

* `mdb_reader_check`, which checks for stale transactions. Unfortunately a reader isn't stale if the owning thread is still alive, so that doesn't work.
* `mdb_reader_list`, which produces a human-readable string containing the lock table contents (used in `mdb_stat` CLI tool). It accepts a callback.

I use the latter, as it's the only way to observe the state I want.

Reply:
I'll add this to the source